### ISSUE TITLE
Please provide a better entry point to pyreverse

### ISSUE THREAD

**Issue by NeilGirdhar** [PyCQA/pylint#5320]
### Current problem

Currently, I have to do:
```python
import pylint

# Set name.
# Collect filenames in files...

sys.argv = 'pyreverse -o png -p'.split() + [name] + files
try:
    pylint.run_pyreverse()
except SystemExit:
    pass  # pylint always does this.
```

### Desired solution

I would like to do something like:
```python
import pylint

pylint.pyreverse(output=pylint.PNG, project=name, files=files)
```

**Comment by Pierre-Sassoulas** [comment]
This is reasonable and should not take too long either.
